wire-protoc-compatibility-tests/src/test/java/com/squareup/wire/StructTest.kt

```diff
@@ -284,17 +241,6 @@ class StructTest {
     assertThat(ProtoAdapter.STRUCT_LIST.decode(googleMessageBytes)).isEqualTo(wireMessage)
   }
 
-  @Test fun emptyList() {
```

Comment:
renamed into `listValueWithoutElements ` to avoid name confusion with Kotlin stdlib.